packages/code-du-travail-frontend/src/common/SurveyModal.js

```diff
@@ -0,0 +1,469 @@
+          />
+          <ActionBar>
+            <Button
+              variant="primary"
+              {...(!state.tailleEntreprise && { disabled: true })}
+              onClick={() => {
+                dispatch({ type: "next" });
+              }}
+            >
+              Suivant
+            </Button>
+            <Button
+              variant="flat"
+              onClick={() => {
+                dispatch({ type: "previous" });
+              }}
+            >
+              Précédent
+            </Button>
+          </ActionBar>
+        </>
+      );
+      break;
+    case 3:
+      content = (
+        <>
+          <Question>
+            Quels sont le(s) contenu(s) que vous souhaiteriez voir
+            enrichi(s)&nbsp;?
+          </Question>
+          <StyledInputCheckbox
+            label="les fiches sur le droit du travail"
+            name="improvements"
+            id="improvements-1"
+            value="fiches droit du travail"
+            onChange={setImprovements}
+            checked={state.improvements.includes("fiches droit du travail")}
+          />
+          <StyledInputCheckbox
+            label="les modèles de documents"
+            name="improvements"
+            id="improvements-2"
+            value="modèles de documents"
+            onChange={setImprovements}
+            checked={state.improvements.includes("modèles de documents")}
+          />
+          <StyledInputCheckbox
+            label="les simulateurs/calculateurs ou autres outils"
+            name="improvements"
+            id="improvements-3"
+            value="outils"
+            onChange={setImprovements}
+            checked={state.improvements.includes("outils")}
+          />
+          <StyledInputCheckbox
+            label="les informations sur une ou des convention(s) collective(s)"
+            name="improvements"
+            id="improvements-4"
+            value="conventions-collectives"
+            onChange={setImprovements}
+            checked={state.improvements.includes("conventions-collectives")}
+          />
+          <StyledInputCheckbox
+            label="d’autre(s) contenu(s)"
+            name="improvements"
+            id="improvements-5"
+            value="autre"
+            onChange={setImprovements}
+            checked={state.improvements.includes("autre")}
+          />
+          {state.improvements.includes("autre") && (
+            <StyledInput
+              name="suggestion"
+              id="suggestion"
+              value={state.suggestion}
+              placeholder="Indiquer le(s) contenu(s) souhaité(s)"
+              onChange={setSuggestion}
+            />
+          )}
+          <StyledInputCheckbox
+            label="je ne sais pas"
+            name="improvements"
+            id="improvements-6"
+            value="nsp"
+            onChange={setImprovements}
+            checked={state.improvements.includes("nsp")}
+          />
+          <ActionBar>
```

Comment:
this could be a component too  (i wonder if wa can reuse the one in tools)

Reply:
Maybe not in this PR ? Next time if we ever reuse it ?

Reply:
Sorry misread, looking at it

Reply:
Seems complicated, tons of logic in the one in the tools plus they are likely to diverge more than converge

Reply:
👍